core/domain/stats_jobs_one_off.py

```diff
@@ -73,6 +73,101 @@ def reduce(exp_id, values):
         yield "%s: %s" % (exp_id, values)
 
 
+class RegenerateMissingStatsModelsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job to regenerate missing stats models for explorations with
+    correct v1 stats from previous versions.
```

Comment:
... for explorations. Note that job XXX MUST be run immediately after this one, otherwise the v2 stats will be inconsistent.

Reply:
Done.